rudder-core/src/main/resources/hooks.d/node-pre-deletion/hook-template

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+# Hooks parameter are passed by environment variable: 
+#
+# - RUDDER_NODEID: the nodeId
+# - RUDDER_NODE_POLICY_SERVER: the node policy server id
+# - RUDDER_NODE_ROLES: a comma separated list of node's server role name
+# - RUDDER_POLICIES_DIRECTORY_CURRENT: the full path of the base directory containing policies for that node
+# - RUDDER_POLICIES_DIRECTORY_NEW: the full path of the base directory containing next policies for that node (during a generation)
+# - RUDDER_POLICIES_DIRECTORY_ARCHIVE: the full path of the base directory containing previous policies for that node
+
+ 
+# Errors code on hooks are interpreted as follow:
+# - 0     : success, no log (appart if debug one)          , continue to next hook
+# - 1-31  : error  , error   log in /var/log/rudder/webapp/, stop processing
+# - 32-63 : warning, warning log in /var/log/rudder/webapp/, continue to next hook
+# - 64-255: reserved for futur use case. Behavior may change without notice. 
```

Comment:
We need to add the 100 return code: 
- 100 : success, warn in /var/log/rudder/webapp/, warn the user in the UI, stop processing /!\ the node won't be deleted